static/webpack.config.js

```diff
@@ -21,12 +21,11 @@ const FixStyleOnlyEntriesPlugin = require("webpack-fix-style-only-entries");
 const config = {
   entry: {
     dev: [__dirname + "/js/dev.ts", __dirname + "/css/dev.scss"],
-    dev_menu: __dirname + "/js/tools/dev_menu.ts",
```

Comment:
was this change intended?

Reply:
it is duplicated in line 26, so removed.